src/vote_stage.rs

```diff
@@ -55,7 +55,7 @@ pub fn create_vote_tx_and_blob(
         blob.meta.set_addr(&addr);
         blob.meta.size = len;
```

Comment:
I think there's a to_blob() function that is exactly these lines...

Reply:
no, that one has a header.  this is just a way to use the blob infrastructure to send packets